src/index.html

```diff
@@ -22,6 +22,442 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="moyo-logo"
+        />
+      </div>
+      <nav class="header__nav">
+        <a
+          href="./index.html"
+          class="header__link is-active"
+        >
+          Apple
+        </a>
+        <a
+          href="./index.html"
+          class="header__link"
+        >
+          Samsung
+        </a>
+        <a
+          href="./index.html"
+          class="header__link"
+        >
+          Smartphones
+        </a>
+        <a
+          href="./index.html"
+          data-qa="nav-hover"
+          class="header__link"
+        >
+          Laptops & Computers
+        </a>
+        <a
+          href="./index.html"
+          class="header__link"
+        >
+          Gadgets
+        </a>
+        <a
+          href="./index.html"
+          class="header__link"
+        >
+          Tablets
+        </a>
+        <a
+          href="./index.html"
+          class="header__link"
+        >
+          Photo
+        </a>
+        <a
+          href="./index.html"
+          class="header__link"
+        >
+          Video
+        </a>
+      </nav>
+    </header>
+
+    <main class="card-container">
+      <card
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__header">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="imac"
+          />
+        </div>
+
+        <div class="card__main">
+          <h1 class="card__product-name">
```

Comment:
This tag used for title of the page, so you need to use the other tag here, fix it everywhere
```suggestion
          <h2 class="card__title">
```